app/MenuBarManager.js

```diff
@@ -116,6 +116,18 @@ class MenuBarManager {
     this._tray = new Tray(MenuBarManager._getTrayIcon());
     this._tray.setToolTip(`Tweet Tray ${app.getVersion()}`);
 
+    if (process.platform === 'linux') {
+      const contextMenu = Menu.buildFromTemplate([
+        {
+          label: 'Compose Tweet',
```

Comment:
If you have the time it should be possible to import and instantiate `LocaleManager` as a variable from here and then access `localeManager.composer.title` from the supported locales.  

If the `app.getLocale()` has been propagated by then, it could be passed to LocaleManager which could be made to invoke `react-localizations`'s `setLanguage()` method. 

`react-localizations` might not work otherwise as I think it's expected to work off the renderer process only.

If you don't think you have the time I can always look into it after the merge. ✌️

Reply:
Yeah, I thought about that. Problem is that the setup happens in the main process, so there's no React running there.

I'll have a look and see if I can figure something out.